### PR TITLE
add forks+flows to metadata obj

### PR DESCRIPTION
**Pull request by ayushkamat**
Signed-off-by: Ayush Kamat <ayush@latch.bio>

### CODE REVIEW

File: latch/types/metadata.py [hunk 1]
```diff
@@ -57,6 +57,71 @@ class LatchAuthor:
...
+        object.__setattr__(self, "flow", list(flow))
+
+
+@dataclass(frozen=True, init=False)
+class Fork(FlowBase):
+    fork: str
+    display_name: str
+    flows: Dict[str, ForkBranch]
+
+    def __init__(self, fork: str, display_name: str, /, **flows: ForkBranch):
```
Comment (kennyworkman) [inline latch/types/metadata.py:119]: whats that weird slash

File: latch/types/metadata.py [hunk 2]
```diff
@@ -101,10 +166,14 @@ class LatchParameter:
     batch_table_column: bool = False
     appearance_type: LatchAppearanceType = LatchAppearanceType.line
     rules: List[LatchRule] = field(default_factory=list)
+    detail: Optional[str] = None
+    _custom_ingestion: Optional[str] = None
 
     def __str__(self):
-        metadata_yaml = yaml.dump(yaml.safe_load(json.dumps(self.dict)))
-        return f"{self.description}\n{metadata_yaml}"
+        metadata_yaml = yaml.safe_dump(self.dict, sort_keys=False)
```
Comment (kennyworkman) [inline latch/types/metadata.py:173]: why not sort keys

**Review by kennyworkman — APPROVED** [review]

File: latch/types/metadata.py (hunk 1 above)
Comment (ayushkamat) [reply latch/types/metadata.py:119]: positional argument delimiter, but its not supported in 3.7 so ill remove it

File: latch/types/metadata.py (hunk 2 above)
Comment (ayushkamat) [reply latch/types/metadata.py:173]: made the resulting yaml easier to read during debugging, not necessary in prod i dont think

**Pull request merged**